### ISSUE TITLE
Extension

### ISSUE THREAD

**Issue by Hedgehogues** [MontFerret/ferret#574]
Hello. I want to use chrome extensions. Does ferret support them?

**Comment by ziflex** [comment]
Hey, right now we do not have any Chrome extensions. What kind of task would you like it to perform?

**Comment by Hedgehogues** [comment]
We want to take the traffic of the requests. By the selenium it is a very difficult do. No, we do the custom selenium extension for this.

**Comment by ziflex** [comment]
So, you would like to record all traffic during browsing a page, correct? 
You have "Network" tab in Chrome DevTools that does the same. Am I missing something?

**Comment by Hedgehogues** [comment]
Yes. I would like to record all traffic. But I want to do this automatically. But DevTool I can use by a hands by a cursor.
